### PR TITLE
Issues/14

### LINKED ISSUE #14: drupal-console settings:check reports some issues
When running any drupal-console command it reports the following check issues:

```
-> % wundertools drupal settings:check   

 The extension mysql is recommended to install.                                                                         

 The extension sqlite3 is recommended to install.                                                                       

 The configuration date.timezone was missing and overwritten with America/Tijuana.                                      
```

Not sure if we care about any of this stuff, but it does make console pretty noisy.

### PR DESCRIPTION
**Pull request by aleksijohansson**
Fixes #14

### CODE REVIEW

**Comment by aleksijohansson** [comment]
@james-nesbitt Is it OK if I assign this PR to you? It would feel a bit silly to assign to myself.

**Comment by james-nesbitt** [comment]
I will take the PR

File: app/.console/phpcheck.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - SimpleXML
+      - SPL
+      - tokenizer
+      - xml
+      - curl
+    recommended:
+      # - mysql
+      - mysqli
+      # - sqlite3
+      - pdo_sqlite
```
Comment (james-nesbitt) [inline app/.console/phpcheck.yml:29]: I wonder if we shouldn't push right to mysqlnd instead of mysqli?

Comment (james-nesbitt) [inline app/.console/phpcheck.yml:29]: not important I think, as we are just talking about console warnings, not implementation.

**Pull request merged**